docroot/sites/library/settings/default.local.settings.php

```diff
@@ -158,3 +158,14 @@
 $settings['trusted_host_patterns'] = [
   '^.+$',
 ];
+
+// Include additional settings files for local development
+$additionalSettingsFiles = [__DIR__ . '/settings.ddev.php'];
+
+foreach($additionalSettingsFiles as $additionalSettingsFile) {
+  if(file_exists($additionalSettingsFile)){
+    require $additionalSettingsFile;
+  }
+}
```

Comment:
Need correct spacing now.
```suggestion
foreach ($additionalSettingsFiles as $additionalSettingsFile) {
  if (file_exists($additionalSettingsFile)) {
    require $additionalSettingsFile;
  }
}
```

Reply:
@pookmish Oops. Okay. it's standardized.